### PR TITLE
Fix doxygen formatting

### LINKED ISSUE #240: Doxygen fails
Check the CI output

### PR DESCRIPTION
**Pull request by niksiboxi**
Resolve #240

### CODE REVIEW

File: src/app_power.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
  */
 /*@{*/
```
Comment (ojousima) [inline src/app_power.h:8]: /** @{ */

File: src/run_integration_tests.h [hunk 2]
```diff
@@ -20,7 +20,7 @@
  */
 /*@{*/
```
Comment (ojousima) [inline src/run_integration_tests.h:21]: /** @( */

File: src/run_integration_tests.h [hunk 3]
```diff
@@ -20,7 +20,7 @@
...
 /**
- * @file run_integration_test.h
+ * \@file run_integration_test.h
```
Comment (ojousima) [inline src/run_integration_tests.h:23]: Is the backslash necessary here? Or is it a symptom of above /*@{*/?

It's my understanding that Doxygen treats both `\` and `@` as "start Doxygen directive", I don't understand why we'd need both.

**Review by ojousima — CHANGES_REQUESTED** [review]

File: src/run_integration_tests.h (hunk 3 above)
Comment (niksiboxi) [reply src/run_integration_tests.h:23]: The doxygen returns a waning:
`warning: the name 'run_integration_test.h' supplied as the argument in the \file statement is not an input file`
 
Current solution is found from Stackoverflow:
https://stackoverflow.com/questions/32872881/markdown-in-doxygen-with-file

Comment (ojousima) [reply src/run_integration_tests.h:23]: 'run_integration_test.h' -> 'run_integration_test**s**.h'

Comment (niksiboxi) [reply src/run_integration_tests.h:23]: Typo: `run_integration_test.h` -> `run_integration_tests.h`

**Review by ojousima — APPROVED** [review]
Thanks

**Pull request merged**